python/pyhive/sqlalchemy_hive.py

```diff
@@ -377,7 +377,7 @@ def get_table_names(self, connection, schema=None, **kw):
         query = 'SHOW TABLES'
         if schema:
             query += ' IN ' + self.identifier_preparer.quote_identifier(schema)
-        return [row[0] for row in connection.execute(text(query))]
+        return [row[1] for row in connection.execute(text(query))]
```

Comment:
this breaks the Hive case.

The PyKyuubi is supposed to work with Kyuubi, Spark Thrift Server, and HiveServer2. In this case, we may need to either switch to the Metadata Thrift API or create different dialects

Reply:
> this breaks the Hive case.
> 
> The PyKyuubi is supposed to work with Kyuubi, Spark Thrift Server, and HiveServer2. In this case, we may need to either switch to the Metadata Thrift API or create different dialects

Thank you for your reply. 
I have made modifications and tests. 
The specific details are in the issue.